sdks/java/io/amazon-web-services2/src/test/java/org/apache/beam/sdk/io/aws2/s3/S3FileSystemTest.java

```diff
@@ -178,8 +178,7 @@ private void testCopy(S3Options options) throws IOException {
     verify(s3FileSystem.getS3Client(), times(1)).copyObject(any(CopyObjectRequest.class));
 
     // we simulate a big object >= 5GB so it takes the multiPart path
-    HeadObjectResponse bigHeadObjectResponse =
-        headObjectResponse.toBuilder().contentLength(5_368_709_120L).build();
+    headObjectResponse.toBuilder().contentLength(5_368_709_120L).build();
```

Comment:
I might be wrong, but isn't the issue here that we should pass te result from this operation into the assert in line 183?

Reply:
I changed it to check the result of the operation into the assert